packages/core/services/utils/getRedisOptions.test.js

```diff
@@ -0,0 +1,39 @@
+import getRedisOptions from './getRedisOptions';
+
+// Set up the mocked cache service.
+describe('cacheService', () => {
+  beforeEach(() => {
+    process.env.REDIS_URL = '';
+    process.env.REDIS_MASTER = '';
+  });
+
+  it('should allow usage of REDIS_URL env var', () => {
+    process.env.REDIS_URL = '127.0.0.1:6379';
```

Comment:
Fwiw this URL doesn't work for me locally, `192.168.50.4:6379` is what works. Since it's just a test I know it doesn't really matter, but it _could_ potentially be confusing if our devs are looking to tests for possible real-world values.

Reply:
The `192.168.50.4` address is what we use for Broadway and is Alley-specific. I *think* that if this is running in production, Redis would be located at the local loopback address. Alternately, I've used a dummy address of `123.456.789.0` before, which is both a real IP and also obviously test data.